### ISSUE TITLE
Some more remaining Settings issues

### ISSUE THREAD

**Issue by sonora** [osmandapp/Osmand#7813]
[1] Configure profile / General settings / No aninations / Apply to all profiles does not work

 [2] Setting 'External input device' is missing the option to revert to 'None'

**Comment by sonora** [comment]
[4] Maybe this in WiP, but how we add the string "Shared: " before each General setting's current status for the selected profile (code ref onBindPreferenceViewHolder) does not make sense. What does it mean to indicate, that this value is also used in other profiles? 

What the user may need to know here is:
- The current **state of the setting for the selected profile** (we have this)
- Perhaps the **profile's default value** for that pref, if any and if deviating, perhaps in the form "Off, profile default: on" or similar

[5] On top of the Coordinate format selection screen we still state "The selected format will be applied throughout the app", but then we allow selection per profile.

[6] When selecting a Driving region other than "Automatic", the status for units of length and speed on their respective settings screens do not reflect any change, not sure this is by design or a bug?

**Comment by sonora** [comment]
[7] When tapping on the **Setting text** (not the toggle to its right) of any setting under Configure profile / General settings / Other (like e.g. the Kaiman filter setting), then change the setting on the **subsequent dialog which shows it description**, then confirm the setting either for all or just the current profile, you are put back on the previos detailed description dialog (instead of it being closed and you get back to the settings screen.)

[8] Setting Configure Profile / General settings / No animations / Apply to all profiles is currently non-functional (but "Apply only to "Current profile" works)

**Comment by sonora** [comment]
[9] Unless I am mistaken, voice announcement is broken in the latest nightlies. It looks like maybe AudioFocus is never requested or the CommandPlayer never gets initialized.

**Comment by sonora** [comment]
**[10]** Our `(Menu button) / Settings` screen contains a section headed 'App profiles'.

In this section a user would expect to be able to
- Configure profiles
- Manage profiles

Instead of presenting the 'Configure profiles' option in clear text, we show the currently active profile. It is not transparent to the users that they could configure any profile by tapping on that one and then chosing a different one on the subsequent screen. We should probaly simply rename the entry to 'Configure profiles'.

**Comment by vshcherb** [comment]
(8) duplicate of (1)
(10) something to think about doesn't look like a critical bug though we have similar bugs with dialogs, so we need to clean up current solution first to be sure we are not fixing after fixes

**Comment by vshcherb** [comment]
(4) Shared is only same for Browse Map profile, so indeed it is going to be reviewed in 3.6. As of now Browse Map value is always considered to be Shared to other profiles unless they have specific value. Though now it functions properly if you change that setting only for Cycling profile for example (it will omit Shared prefix)

**Comment by sonora** [comment]
Yes, apologies about [8], my oversight, looks like I am overworked. :)

Regarding [10], it is a simple wording issue, very easy to fix.

Regarding [4]: Ah, I undetstand: So some profiles are sort of 'parents' to other profiles, and changing some of their settings affects the children as well. But this is a concept not easily understood, I wonder if it is really worth the hassle of implemting and then user-supporting it...  

Remember our reputation is we are overly complicated, I guess with every release we should also try to create transparency to a degree (which release 3.5 x with its settings re-arangements could actually achieve if we do it right now).

**[11]** We should e.g. also drasticallly reduce the Drawer now by cleaning out everything which has logically moved to 'Configure profile'.

**Comment by Chumva** [comment]
1 - do_not_use_animations should be made to general preference dependent on browse map profile (similar preferences also in trip recording now)
2 - It sets this option if the toggle is off

Previously all profile dependent preferences were dependent on browse map profile, now only those in trip recording settings and general settings screens.
Changes in the trip recording - #7821
Check for default browse map value -https://github.com/osmandapp/Osmand/blob/master/OsmAnd/src/net/osmand/plus/OsmandSettings.java#L635

**Comment by vshcherb** [comment]
[11] Sounds like a plan for next release
[10] It is not just wording issue, it also shows an intention, if user wants to configure 1 profile, though in that dialog user can change profile itself and start configuring another one

**Comment by sonora** [comment]
STATUS:
- [x] [1]
- [ ] [2] Still not fixed, from what I see
- [x] [3]
- [ ] [4] I feel we could do better here. Most users will not understand what it means that we state "Shared". Remember that 99% of users will work with the base profiles only.
- [x] [5]
- [ ] [6] Still not fixed, from what I see
- [x] [7]
- [x] [8]
- [ ] [9] Still not fixed. (Selecting "Sound off" on the route config dialog will mute all voice prompts, but the toggle "Voice prompts" under the profile's Navigation Settings will remain on. The two must be synced.)
- [ ] [10] Still open. If we want to make this more than a wording issue and also facilitate changing profiles on this OsmAnd settings screen, that's fine: Then we should simply use the identical pattern we already have at the top of the Drawer, i.e. show the current profile with the little selection arrow to its right, and below it show a second line "Configure profile".  Creating all sorts of different looking mechanisms in different places creates complication and confusion.
- [ ] [11] We have drastically added new config screens and dialogs, in an effort to make our structure more logical. If we do not use the same release to throw out the old way of having it, we seriously miss our chance to make the app more likeable ... ;)
